src/MetricsProvider.ts

```diff
@@ -1,21 +1,23 @@
 import { COUNTLY_SETUP_DEFAULTS } from './config.js'
+import { EventAccumulator } from './EventAccumulator.js'
 
-import type { metricFeatures, CountlyWebSdk } from 'countly-sdk-web'
 import type { CountlyNodeSdk } from 'countly-sdk-nodejs'
+import type { CountlyWebSdk, metricFeatures } from 'countly-sdk-web'
 import type { consentTypes, consentTypesExceptAll } from './types/index.js'
```

Comment:
you should update eslint so it's organizing imports instead of depending on your editor for that! otherwise they'll keep getting out of order ;)

speaking of ordering imports. Can we group the imports? we've currently got two relative imports at the top, then 2 3p imports (countly-sdk-*), then another relative.